server/events/command_runner.go

```diff
@@ -314,13 +316,25 @@ func (c *DefaultCommandRunner) automerge(ctx *CommandContext, pullStatus models.
 func (c *DefaultCommandRunner) runProjectCmds(cmds []models.ProjectCommandContext, cmdName models.CommandName) CommandResult {
 	var results []models.ProjectResult
 	for _, pCmd := range cmds {
+		cmdStart := time.Now().UnixNano()
```

Comment:
It's going to get ugly adding this for all the different metrics people will want measured. Can you make this cleaner?